x-pack/plugins/cases/server/services/cases/index.ts

```diff
@@ -517,7 +565,10 @@ export class CasesService {
         { id, references: transformedAttributes.referenceHandler.build(), refresh }
       );
 
-      return transformSavedObjectToExternalModel(createdCase);
+      const res = transformSavedObjectToExternalModel(createdCase);
+      decodeOrThrow(CaseTransformedAttributesRt)(res.attributes);
```

Comment:
Do we need to return the result here? Otherwise we could have extra properties in the response right?

Reply:
Good catch!